### PR TITLE
Fixed error when call function disable/enable control button before initializing the control

### LINKED ISSUE #1141: setButtonDisabled not working
Good afternoon, yesterday everything worked fine, the code did not change, but the error pops up. here is part of the code:
```
var optionsControl = {
        position: 'topleft', // toolbar position, options are 'topleft', 'topright', 'bottomleft', 'bottomright'
        drawMarker: true,  // adds button to draw markers
        drawPolygon: false,  // adds button to draw a polygon
        drawPolyline: false,  // adds button to draw a polyline
        drawCircle: false,  // adds button to draw a cricle
        editPolygon: false,  // adds button to toggle global edit mode
        deleteLayer: true,   // adds a button to delete layers
		cutPolygon: false,
		rotateMode: false,
		drawRectangle: false,
		drawCircleMarker: false,
    };

map.pm.Toolbar.setButtonDisabled('drawMarker', true);
```
If you remove the line `map.pm.Toolbar.setButtonDisabled('drawMarker', true);`, then there is no error, but this function is necessary. The error is like this:
Uncaught TypeError: Cannot read properties of undefined (reading 'children')                                              leaflet-geoman.min.js:1 
    at i._updateDisabled (leaflet-geoman.min.js:1:67669)
    at i.disable (leaflet-geoman.min.js:1:64799)
    at i.setButtonDisabled (leaflet-geoman.min.js:1:79813)
    at (индекс):2717:16

### PR DESCRIPTION
**Pull request by vvlladd28**
Fixed #1141

### CODE REVIEW

**Comment by vvlladd28** [comment]
Please review my PR @Falke-Design

**Comment by Falke-Design** [comment]
Can you please add a test?

**Comment by vvlladd28** [comment]
Can you tell me how to write a test correctly because all tests are called after initializing control?

**Comment by Falke-Design** [comment]
Look into this [test](https://github.com/geoman-io/leaflet-geoman/blob/9297f87f29c98978fb6f70980b7327bfa3098202/cypress/integration/circle.spec.js#L187). First remove the existing map, then create a new map as [here](https://github.com/geoman-io/leaflet-geoman/blob/9297f87f29c98978fb6f70980b7327bfa3098202/cypress/support/index.js#L37)

**Comment by vvlladd28** [comment]
Thanks for the help in write test

File: cypress/integration/toolbar.spec.js [hunk 1]
```diff
@@ -449,4 +449,36 @@ describe('Testing the Toolbar', () => {
...
+    cy.window().then(({ map, L }) => {
+      map.remove();
+
+      const tiles = L.tileLayer(
+        'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+        {
+          attribution:
+            '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+        }
+      );
```
Comment (Falke-Design) [inline cypress/integration/toolbar.spec.js:462]:
```suggestion
```

tiles are not necessary

File: cypress/integration/toolbar.spec.js [hunk 2]
```diff
@@ -449,4 +449,36 @@ describe('Testing the Toolbar', () => {
...
+
+      // create the map
+      map = L.map('map', {
+        preferCanvas: false,
+        doubleClickZoom: false, // Leaflet 1.8 DoubleTap fix
+      })
+        .setView([51.505, -0.09], 13)
+        .addLayer(tiles);
+
+      cy.window.map = map;
```
Comment (Falke-Design) [inline cypress/integration/toolbar.spec.js:472]:
```suggestion
```
we don't want to overwrite the global map object

**Review by Falke-Design — CHANGES_REQUESTED** [review]

**Review by Falke-Design — APPROVED** [review]

**Pull request merged**